tfgcv/planned_assets.go

```diff
@@ -67,7 +51,47 @@ func ReadPlannedAssets(path, project, ancestry string) ([]google.Asset, error) {
 		return nil, errors.Wrap(err, "building google converter")
 	}
 
-	for _, r := range tfplan.ComposeResources(plan, converter.Schemas()) {
+	var resources []tfplan.Resource
+
+	switch tfVersion {
```

Comment:
We should error out on the default case here since we expect to be given a valid tfVersion.

Reply:
done